modules/optimizing-the-MTU-for-your-network.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// networking/network-optimization.adoc
+
+[id='optimizing-the-mtu-for-your-network-{context}']
+= Optimizing the MTU for your network
+
```

Comment:
I feel like we're missing a little bit of context in this section as well. Why would you need to set the MTU? What is the use case?

Reply:
By default, with the latest automatic detection of MTUs with OCP 4.0 one does not need to tweak MTU settings but in-case one wants to have a fixed MTU size for their workloads then one can do so by tweaking the above settings.